lib/oauth.rb

```diff
@@ -2,7 +2,7 @@ module Oauth
   SCOPES = %w[read_prefs write_prefs write_diary write_api read_gpx write_gpx write_notes].freeze
   PRIVILEGED_SCOPES = %w[read_email skip_authorization].freeze
   MODERATOR_SCOPES = %w[write_redactions].freeze
-  OAUTH2_SCOPES = %w[write_redactions openid].freeze
+  OAUTH2_SCOPES = %w[write_redactions message_send message_consume openid].freeze
```

Comment:
Would it be more consistent to have something like 'write_messages', 'read_messages'?

Reply:
To clarify, I don't mind the names and/or having different set of scopes.
- read message without modifying anything (peek messages)
- mark messages read or unread
- mark messages as deleted
- send messages to other users

It seemed as if the reading messages makes sense together with the ability to mark them as read and deleted, and I named the scope for those ops `message_consume`

Reply:
@tordans Would `cosume_messages` and `send_messages` be better?

I've just noticed that all other scopes are all in _verb_noun_ form.